frontend/javascripts/oxalis/view/settings/setting_input_views.js

```diff
@@ -338,20 +341,26 @@ export class UserBoundingBoxInput extends React.PureComponent<UserBoundingBoxInp
   render() {
     const { name } = this.state;
     const tooltipStyle = this.state.isValid ? null : { backgroundColor: "red" };
-    const { tooltipTitle, color, isVisible, onDelete } = this.props;
+    const { tooltipTitle, color, isVisible, onDelete, onExport } = this.props;
     const upscaledColor = ((color.map(colorPart => colorPart * 255): any): Vector3);
     const iconStyle = { margin: "auto 0px auto 6px" };
+    const exportButton = features().jobsEnabled ? (
+      <Tooltip title="Export data from this bouding box.">
+        <Icon type="download" onClick={onExport} style={iconStyle} />
+      </Tooltip>
+    ) : null;
     return (
       <React.Fragment>
         <Row style={{ marginBottom: 16 }}>
-          <Col span={22}>
+          <Col span={20}>
             <Switch
               size="small"
               onChange={this.handleVisibilityChange}
               checked={isVisible}
               style={{ margin: "auto 0px" }}
             />
           </Col>
+          <Col span={2}>{exportButton}</Col>
```

Comment:
this allocates space for the button even if it isn't rendered. I'd put the column into the variable (and rename it to `exportColumn`). the `span` property of the first `Col` element could be switched between 20 or 22 depending on `exportColumn != null`.